tsconfig.json

```diff
@@ -4,10 +4,12 @@
     "paths": {
       "@final-ui/app": ["./packages/playground/src"],
       "@final-ui/react": ["./packages/react/src"],
+      "@final-ui/react/jsx-runtime": ["./packages/react/src/jsx"],
       "@final-ui/ws-client": ["./packages/ws-client/src"],
       "@final-ui/ws-server": ["./packages/ws-server/src"],
       "@final-ui/tamagui": ["./packages/tamagui/src"],
-      "@final-ui/kit": ["./packages/kit/src"]
+      "@final-ui/kit": ["./packages/kit/src"],
```

Comment:
by default, `ui/kit` is a server package and to load it on the client, you need to import all packages from the `runtime`.

Reply:
That configuration is only specific to our dev setup. In production, developers will be automatically getting this via "exports" field.